substrate/frame/revive/src/lib.rs

```diff
@@ -88,7 +91,7 @@ pub use weights::WeightInfo;
 pub use crate::wasm::SyscallDoc;
 
 type TrieId = BoundedVec<u8, ConstU32<128>>;
-type BalanceOf<T> =
+pub type BalanceOf<T> =
```

Comment:
I don't think this is used anywhere? Even if so I don't think we should export those short hand aliases.